### ISSUE TITLE
Compilation issues with BeaconManager JavaDoc example code

### ISSUE THREAD

**Issue by maass-bezirk** [Estimote/Android-SDK#166]
From https://estimote.github.io/Android-SDK/JavaDocs/com/estimote/sdk/BeaconManager.html:

```java
   private static final String ESTIMOTE_PROXIMITY_UUID = UUID.fromString("B9407F30-F5F8-466E-AFF9-25556B57FE6D");
   private static final Region ALL_ESTIMOTE_BEACONS = new Region("rid", ESTIMOTE_PROXIMITY_UUID, null, null)
```

`ESTIMOTE_PROXIMITY_UUID` should be a `UUID` not a `String` and the second line has no semi-colon.

**Comment by pawelDylag** [comment]
Hello @maass-bezirk !

Thanks for reporting that - it it our mistake indeed. Will fix that soon :)

Have a nice day!

**Comment by pawelDylag** [comment]
Fixed in 0.12.0
